pkg/rpcclient/rpc_test.go

```diff
@@ -1938,7 +1938,7 @@ func initTestServer(t *testing.T, resp string) *httptest.Server {
 			ws, err := upgrader.Upgrade(w, req, nil)
 			require.NoError(t, err)
 			for {
-				err = ws.SetReadDeadline(time.Now().Add(2 * time.Second))
+				err = ws.SetReadDeadline(time.Now().Add(5 * time.Second))
```

Comment:
There is an option to set deadline to 0 (no deadline). I'm starting to think that it's not bad for our tests.